deb/default.json

```diff
@@ -0,0 +1,849 @@
+				"removexattr",
+				"rename",
+				"renameat",
+				"renameat2",
+				"restart_syscall",
+				"rmdir",
+				"rseq",
+				"rt_sigaction",
+				"rt_sigpending",
+				"rt_sigprocmask",
+				"rt_sigqueueinfo",
+				"rt_sigreturn",
+				"rt_sigsuspend",
+				"rt_sigtimedwait",
+				"rt_sigtimedwait_time64",
+				"rt_tgsigqueueinfo",
+				"sched_getaffinity",
+				"sched_getattr",
+				"sched_getparam",
+				"sched_get_priority_max",
+				"sched_get_priority_min",
+				"sched_getscheduler",
+				"sched_rr_get_interval",
+				"sched_rr_get_interval_time64",
+				"sched_setaffinity",
+				"sched_setattr",
+				"sched_setparam",
+				"sched_setscheduler",
+				"sched_yield",
+				"seccomp",
+				"select",
+				"semctl",
+				"semget",
+				"semop",
+				"semtimedop",
+				"semtimedop_time64",
+				"send",
+				"sendfile",
+				"sendfile64",
+				"sendmmsg",
+				"sendmsg",
+				"sendto",
+				"setfsgid",
+				"setfsgid32",
+				"setfsuid",
+				"setfsuid32",
+				"setgid",
+				"setgid32",
+				"setgroups",
+				"setgroups32",
+				"setitimer",
+				"setpgid",
+				"setpriority",
+				"setregid",
+				"setregid32",
+				"setresgid",
+				"setresgid32",
+				"setresuid",
+				"setresuid32",
+				"setreuid",
+				"setreuid32",
+				"setrlimit",
+				"set_robust_list",
+				"setsid",
+				"setsockopt",
+				"set_thread_area",
+				"set_tid_address",
+				"setuid",
+				"setuid32",
+				"setxattr",
+				"shmat",
+				"shmctl",
+				"shmdt",
+				"shmget",
+				"shutdown",
+				"sigaltstack",
+				"signalfd",
+				"signalfd4",
+				"sigprocmask",
+				"sigreturn",
+				"socketcall",
+				"socketpair",
+				"splice",
+				"stat",
+				"stat64",
+				"statfs",
+				"statfs64",
+				"statx",
+				"symlink",
+				"symlinkat",
+				"sync",
+				"sync_file_range",
+				"syncfs",
+				"sysinfo",
+				"tee",
+				"tgkill",
+				"time",
+				"timer_create",
+				"timer_delete",
+				"timer_getoverrun",
+				"timer_gettime",
+				"timer_gettime64",
+				"timer_settime",
+				"timer_settime64",
+				"timerfd_create",
+				"timerfd_gettime",
+				"timerfd_gettime64",
+				"timerfd_settime",
+				"timerfd_settime64",
+				"times",
+				"tkill",
+				"truncate",
+				"truncate64",
+				"ugetrlimit",
+				"umask",
+				"uname",
+				"unlink",
+				"unlinkat",
+				"utime",
+				"utimensat",
+				"utimensat_time64",
+				"utimes",
+				"vfork",
+				"vmsplice",
+				"wait4",
+				"waitid",
+				"waitpid",
+				"write",
+				"writev"
+			],
+			"action": "SCMP_ACT_ALLOW"
+		},
+		{
+			"names": [
+				"process_vm_readv",
+				"process_vm_writev",
+				"ptrace"
+			],
+			"action": "SCMP_ACT_ALLOW",
+			"includes": {
+				"minKernel": "4.8"
+			}
+		},
+		{
+			"names": [
+				"fchmodat2",
+				"map_shadow_stack"
+			],
+			"action": "SCMP_ACT_ALLOW",
+			"includes": {
+				"minKernel": "6.6"
+			}
+		},
+		{
+			"names": [
+				"futex_requeue",
+				"futex_wait",
+				"futex_wake"
+			],
+			"action": "SCMP_ACT_ALLOW",
+			"includes": {
+				"minKernel": "6.7"
+			}
+		},
```

Comment:
Trying if this would do the trick; conditionally add the new syscalls based on kernel version (not sure if it works though)

Reply:
OK, looks like this won't fix the issue. I was hoping that making these conditional would exclude them from the seccomp-profile, and make use of the stubs that runc creates, but that's not the case;


```
tar: engine/cli/winresources: Cannot change mode to rwxrwxr-x: Operation not permitted
tar: engine/cli: Cannot change mode to rwxrwxr-x: Operation not permitted
tar: engine/integration-cli/fixtures/https/client-key.pem: Cannot change mode to rwxrwxrwx: Operation not permitted
tar: engine/integration-cli/fixtures/https/client-cert.pem: Cannot change mode to rwxrwxrwx: Operation not permitted
tar: engine/integration-cli/fixtures/https/server-cert.pem: Cannot change mode to rwxrwxrwx: Operation not permitted
tar: engine/integration-cli/fixtures/https/server-key.pem: Cannot change mode to rwxrwxrwx: Operation not permitted
tar: engine/integration-cli/fixtures/https/ca.pem: Cannot change mode to rwxrwxrwx: Operation not permitted
tar: engine/integration-cli/fixtures/https: Cannot change mode to rwxrwxr-x: Operation not permitted
tar: engine/integration-cli/fixtures: Cannot change mode to rwxrwxr-x: Operation not permitted
tar: engine/integration-cli: Cannot change mode to rwxrwxr-x: Operation not permitted
tar: Exiting with failure status due to previous errors
make[1]: *** [Makefile:85: ubuntu-noble] Error 2
make[1]: Leaving directory '/home/ubuntu/workspace/docker-ce-packaging_PR-1015/deb'
make: *** [Makefile:83: ubuntu-noble] Error 2
script returned exit code 2
```